src/mixins/payment-mixin.ts

```diff
@@ -536,7 +536,7 @@ export class PaymentMixin extends Mixins(ActionMixin) {
       if (!paymentResponse) throw new Error('Got error from getNameRequestPayment()')
 
       // eslint-disable-next-line @typescript-eslint/no-unused-vars
-      const { payment, sbcPayment = { receipts: [], status_code: '' }, statusCode, completionDate } = paymentResponse
+      const { payment, sbcPayment = { receipts: [], status_code: '' }, statusCode, completionDate } = paymentResponse[0]
```

Comment:
I think adding this, makes this show again:

![image](https://github.com/bcgov/namerequest/assets/3484109/ed3926f3-6bbd-4869-a4da-9b5554289bc2)

Reply:
maybe changing this to:

```
      let resp = paymentResponse
      if (resp instanceof Array) {
        resp = resp[0]
      }

      // eslint-disable-next-line @typescript-eslint/no-unused-vars
      const { payment, sbcPayment = { receipts: [], status_code: '' }, statusCode, completionDate } = resp
```
will be safer, in case there is a None value

Reply:
I've removed this section of code, I doubt it was called. I checked on TEST it wasn't called.

Reply:
already a check above it @bolyachevets       if (!paymentResponse) throw new Error('Got error from getNameRequestPayment()')

Reply:
Too risky, put this this back to what it was, which was unfortunately broken